app/Console/Commands/SendExpectedCheckinAlerts.php

```diff
@@ -44,14 +44,14 @@ public function fire()
     {
 
         $whenNotify = Carbon::now()->addDays(7);
-        $assets = Asset::with('assigneduser')->whereNotNull('expected_checkin')->where('expected_checkin', '<=', $whenNotify)->get();
+        $assets = Asset::with('assignedTo')->whereNotNull('expected_checkin')->where('expected_checkin', '<=', $whenNotify)->get();
```

Comment:
What's the reason for changing this? assigneduser should have worked (and did in my tests)

Reply:
the assigneduser relationship as existed was a lie.  it found the user that matched the assigned_to id, reguardless of the assigned_type.  I tried to modify it to only return something when checked out to a user, but it caused a lot of issues when the relationship returned null.